### PR TITLE
Updating homelessSituationType and homelessnessRiskSituationType enums

### PR DESCRIPTION
**Pull request by bastosmichael**
## Description of change
We needed to add standard enums to the homelessness section of the 526 because previously it was just considered an open field. Also added homelessnessRisk factors.

## Original issue(s)
https://vajira.max.gov/browse/API-1081

### CODE REVIEW

File: modules/claims_api/config/schemas/526.json [hunk 1]
```diff
@@ -294,7 +293,33 @@
               ],
               "properties": {
                 "homelessSituationType": {
-                  "type": "string"
+                  "type": "string",
+                  "default": "OTHER",
+                  "enum": ["FLEEING_CURRENT_RESIDENCE",
```
Comment (charleystran) [inline modules/claims_api/config/schemas/526.json:298]: Correct me if I am wrong here, but I thought the point of this was to use the translation keys here https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/disability_compensation_form/data_translation_all_claim.rb#L13 and not the values?

File: modules/claims_api/spec/fixtures/form_526_evss.json [hunk 2]
```diff
@@ -43,13 +43,21 @@
         ]
```
Comment (edmangimelli) [inline modules/claims_api/spec/fixtures/form_526_evss.json:43]: on line 26 of this file, the value "Air Force Reserve" is being used.
that should be "Air Force Reserves" (plural) to match the enum

Comment (edmangimelli) [reply modules/claims_api/spec/fixtures/form_526_evss.json:43]: also, this fixture is missing the "standardClaim" field, which is required by modules/claims_api/config/schemas/526.json

File: modules/claims_api/spec/fixtures/form_526_evss.json [hunk 3]
```diff
@@ -43,13 +43,21 @@
...
+    "homelessness": {
+      "pointOfContact": {
+        "pointOfContactName": "John Doe",
+        "primaryPhone": "555-555-5555"
+      },
+      "currentlyHomeless": {
+        "homelessSituationType": "FLEEING_CURRENT_RESIDENCE",
+        "otherLivingSituation": "other living situations"
+      }
+    },
```
Comment (edmangimelli) [inline modules/claims_api/spec/fixtures/form_526_evss.json:55]: the "homelessness" object needs to be moved under the "veteran" top-level object. that's why it looks like it's not obeying the `"required": [ "pointOfContact" ],` constraint

File: modules/claims_api/spec/fixtures/form_526_json_api.json [hunk 4]
```diff
@@ -16,6 +16,16 @@
...
+      "homelessness": {
+        "pointOfContact": {
+          "pointOfContactName": "John Doe",
+          "primaryPhone": "555-555-5555"
+        },
+        "currentlyHomeless": {
+          "homelessSituationType": "FLEEING_CURRENT_RESIDENCE",
+          "otherLivingSituation": "other living situations"
+        }
+      },
```
Comment (edmangimelli) [inline modules/claims_api/spec/fixtures/form_526_json_api.json:28]: same situation. needs to be moved under "veteran"

File: modules/claims_api/spec/fixtures/form_526_evss.json [hunk 5]
```diff
@@ -43,13 +43,21 @@
         ]
       }
     ],
+    "homelessness": {
+      "pointOfContact": {
+        "pointOfContactName": "John Doe",
+        "primaryPhone": "555-555-5555"
```
Comment (edmangimelli) [inline modules/claims_api/spec/fixtures/form_526_evss.json:49]:
```suggestion
          "primaryPhone": {
            "areaCode": "555",
            "phoneNumber": "555-5555"
          }
```

File: modules/claims_api/spec/fixtures/form_526_json_api.json [hunk 6]
```diff
@@ -16,6 +16,16 @@
         }
       },
       "claimantCertification": true,
+      "homelessness": {
+        "pointOfContact": {
+          "pointOfContactName": "John Doe",
+          "primaryPhone": "555-555-5555"
```
Comment (edmangimelli) [inline modules/claims_api/spec/fixtures/form_526_json_api.json:22]:
```suggestion
          "primaryPhone": {
            "areaCode": "555",
            "phoneNumber": "555-5555"
          }
```

File: modules/claims_api/spec/fixtures/form_526_json_api.json (hunk 4 above)
Comment (bastosmichael) [reply modules/claims_api/spec/fixtures/form_526_json_api.json]: Thanks for catching this.

File: modules/claims_api/config/schemas/526.json (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/config/schemas/526.json]: Fixed

File: modules/claims_api/spec/fixtures/form_526_evss.json (hunk 3 above)
Comment (bastosmichael) [reply modules/claims_api/spec/fixtures/form_526_evss.json]: Fixed

**Pull request merged**